Source/WebCore/dom/ViewTransition.h

```diff
@@ -104,6 +107,13 @@ struct OrderedNamedElementsMap {
         return nullptr;
     }
 
+    const CapturedElement* find(const AtomString& key) const
```

Comment:
We should return a `std::optional<CapturedElement>` or `const std::optional<CapturedElement>&` but not a pointer.

Reply:
Is it how find methods are typically defined in WTF?

Reply:
WTF uses iterators

Reply:
But well a pointer could be considered a kind of iterator. I'm fine with this :)